great_expectations/expectations/registry.py

```diff
@@ -399,6 +399,9 @@ def get_expectation_impl(expectation_name: str) -> Type[Expectation]:
         )
         expectation_name = renamed[expectation_name]
 
+    if not _registered_expectations:
+        _register_core_expectations()
```

Comment:
Move this logic closer to where it is used - if we try to access `_registred_expectations`, this should be run before